source/server/admin/config_dump_handler.cc

```diff
@@ -282,6 +282,11 @@ absl::optional<std::pair<Http::Code, std::string>> ConfigDumpHandler::addAllConf
     auto* config = dump.add_configs();
     config->PackFrom(*message);
   }
+  if (dump.configs().empty() && mask.has_value()) {
```

Comment:
can you just use `config` here rather than calling `dump->configs()`?

Reply:
No, `config` is out of scope at this point.

Reply:
indeed :)